x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -99,20 +97,29 @@ function getSpanItem(span: Span): IWaterfallSpan {
   };
 }
 
-function getErrorItem(error: APMError): IWaterfallError {
+function getErrorItem(
+  error: APMError,
+  items: IWaterfallItem[],
+  entryWaterfallTransaction?: IWaterfallTransaction
+): IWaterfallError {
+  const entryTimestamp = entryWaterfallTransaction?.doc.timestamp.us ?? 0;
+  const parent = items.find(
+    waterfallItem => waterfallItem.id === error?.parent?.id
+  );
   return {
     docType: 'error',
     doc: error,
     id: error.error.id,
-    parentId: error.parent?.id,
-    offset: 0,
-    skew: 0,
+    parent,
+    parentId: parent?.id,
+    offset: error.timestamp.us - entryTimestamp,
+    skew: getClockSkew((error as unknown) as IWaterfallItem, parent),
```

Comment:
I think that's the reason for the error I posted, `error` isn't an WaterfallItem, so it doesn't have `docType`.
<img width="1294" alt="Screenshot 2020-01-28 at 13 23 57" src="https://user-images.githubusercontent.com/55978943/73265620-9c0a9480-41d5-11ea-91a1-04f861b5c6a0.png">